migrations/20240624145325_recreate_auth_functions.up.sql

```diff
@@ -0,0 +1,52 @@
+-- set the search_path to an empty string to force fully qualified names in the function 
+do $$ 
+begin
+    -- auth.uid() function
+    create or replace function {{ index .Options "Namespace" }}.uid() 
+        returns uuid 
+        set search_path to '' 
+    as $func$
+        select nullif(
```

Comment:
Should use this: https://github.com/supabase/auth/blob/e81c25d19551fdebfc5197d96bc220ddb0f8227b/migrations/20220224000811_update_auth_functions.up.sql#L25

instead